ads/google/imaVideo.js

```diff
@@ -1138,13 +1138,17 @@ export function onPlayPauseClick() {
  * @visibleForTesting
  */
 export function playVideo() {
-  setStyle(adContainerDiv, 'display', 'none');
+  if (adsActive) {
```

Comment:
the check for adsActive has been moved from onMessage event https://github.com/ampproject/amphtml/pull/23660/files#diff-64400de024de2e5299df7084028aabd9R1425